api/workAllocation/locationController.ts

```diff
@@ -15,13 +15,14 @@ export async function getLocationById(req: EnhancedRequest, res: Response, next:
 
   try {
 
-    const path: string = prepareGetLocationByIdUrl(baseUrl, req.params.locationId)
-    const locationById = await handleLocationGet(path, req)
+    const path: string = prepareGetLocationByIdUrl(baseUrl, req.params.locationId);
+    /*TODO: Implement get location*/
+    const locationById = await handleLocationGet(path, req);
```

Comment:
No, we're actually receiving constant locations that are defined within the node layer

Reply:
Hmmm... but I would assume _this_ bit of code is going to remain just as it is and that the internals of `handleLocationGet()` will change to return locations from an endpoint, rather than hardcoded ones...?